File: online-judge-solutions/Codeforces/421A.py
#https://codeforces.com/problemset/problem/421/A
#Lang		:	Python 3.8
#Time		:	46 ms
#Memory         :	0 KB
n,a,b=map(int,input().split())
v=list(map(int,input().split()))
w=list(map(int,input().split()))
m={c:2 for c in w}
m.update({c:1 for c in v})
m=dict(sorted(m.items()))
print(*list(m.values()))

File: online-judge-solutions/Codeforces/421A.cpp
//https://codeforces.com/problemset/problem/421/A
//Lang		:	GNU C++14
//Time		:	15 ms
//Memory	:	100 KB
#include<iostream>
#include<vector>
#include<map>
using namespace std;
int main(){
	int n,a,b,x;
	while(cin>>n>>a>>b){
		vector<int> v(a),w(b);
		map<int,int> m;
		for(int i=0;i<a;i++){cin>>v[i];}
		for(int i=0;i<b;i++){
			cin>>w[i];
			m[w[i]]=2;
		}
		for(int i=0;i<a;i++){m[v[i]]=1;}
		for(auto i=m.begin();i!=m.end();i++){
			cout<<i->second<<" ";
		}cout<<endl;
	}
	return 0;
}
